src/public/media.ts

```diff
@@ -437,6 +438,12 @@ export namespace media {
     }
     ensureInitialized(FrameContexts.content, FrameContexts.task);
 
+    if (!GlobalVars.isFramelessWindow) {
```

Comment:
This is not a good way to check for web/desktop. Please look at the `GlobalVars.hostClientType` instead or better yet implement a handler in the web/desktop client code itself to return the error.

Reply:
Sure. I have modified the code to use GlobalVars.hostClientType.